proposals/NNNN-initial-and-hint-text-for-keyboard.md

```diff
@@ -0,0 +1,51 @@
+# Arguments initialText and hintText for SDL Keyboard
+
+* Proposal: [SDL-NNNN](NNNN-initial-and-hint-text-for-keyboard.md)
+* Author: [MarekForys](https://github.com/mforys) (TomTom)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core, Android, iOS]
+
+
+## Introduction
+
+SDl Keyboard may be started with argument initialText, but it's displayed in iOS version as blue hint/description in the input field of SDL Keyboard. In the case of Android this initial text is never displayed.
+
+There should be a possibility of using initial and hint/description text as well.
+All available documentation is here: [SDLPerformInteraction](https://smartdevicelink.com/en/docs/iOS/master/Classes/SDLPerformInteraction/).
+
+
+## Motivation
+
+Sometimes user wants to continue to input a text which partially is already filled in device's input text field, then after connecting to SDL the text field is totally empty. Another scenario is when some text is filled into SDl keyboard then we press Enter and we launch Keyboard again. Input text field is blank and previously entered text is gone.
+
+
+## Proposed solution
+
+When creating an SDL Keyboard we should give two text arguments: initialText and hintText. Argument hintText should just replace current initialText. Flow for initialText must be developed in all submodules from the scratch.
+
+So the perfect description of these arguments should be like that:
```

Comment:
Please update from "like that:" to "like this:"

Reply:
Done